collect_app/src/main/res/values/strings.xml

```diff
@@ -661,7 +661,7 @@
     <string name="location_providers_disabled_dialog_message">This form wants to track your location but location providers are disabled. Please enable in Android settings.</string>
     <string name="go_to_settings">Go to settings</string>
     <string name="track_location">Track location</string>
-    <string name="background_location_disabled">This form wants to track your location but tracking is disabled. Please enable in the \u0020\u0020⋮\u0020\u0020 menu above.</string>
-    <string name="background_location_enabled">This form tracks your location. You can disable tracking in the \u0020\u0020⋮\u0020\u0020 menu above.</string>
+    <string name="background_location_disabled">This form wants to track your location but tracking is disabled. Please enable in the %1$s menu above.</string>
+    <string name="background_location_enabled">This form tracks your location. You can disable tracking in the %1$s menu above.</string>
```

Comment:
I think this will result in double spaces for Android 4. @huangyz0918, can you please confirm, and if so, fix.